### PR TITLE
Add php8.1 and php8.2 image options

### PR DESCRIPTION
**Pull request by hsiemon-globant**

### CODE REVIEW

**Comment by hsiemon-globant** [comment]
@alwaysblank Can you review this one pls?

**Review by alwaysblank — APPROVED** [review]
I'm not in a position to test this right now, but it looks fine to me. 

I should point out, though, that I don't work for Human Made any more, so I don't have write access and can't merge this, so you might want to ping another contributor to get that done.

**Review by nexstar-kadamwhite — APPROVED** [review]

**Review by kadamwhite — APPROVED** [review]

**Pull request merged**
